modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -73,8 +82,8 @@ func newSendTxCmd() *cobra.Command {
 		Use:   "send-tx [connection-id] [path/to/packet_msg.json]",
 		Short: "Send an interchain account tx on the provided connection.",
 		Long: strings.TrimSpace(`Submits pre-built packet data containing messages to be executed on the host chain 
-and attempts to send the packet. Packet data is provided as json, file or string. An 
-appropriate relative timeoutTimestamp must be provided with flag {relative-packet-timeout}`),
+and attempts to send the packet. Packet data is provided as json, file or string. A relative timeout timestamp can be provided with flag {packet-timeout-timestamp}.
```

Comment:
If it's correct that absolute timeout timestamps can be used when `flagAbsoluteTimeouts` is true and `flagPacketTimeoutTimestamp` is also set, should we update this doc string to clarify that? At the moment, the way it reads I think a reader would think `flagPacketTimeoutTimestamp` can only be used for relative timeouts...

Reply:
Updated in https://github.com/cosmos/ibc-go/pull/5908/commits/7f673fb86238c22d85d464de0195f6e94f970765

feel free to push another commit if you want to change it.